app/services/variant_units/variant_and_line_item_naming.rb

```diff
@@ -65,6 +65,7 @@ def update_units
     def unit_value_attributes
       units = { unit_presentation: option_value_name }
       units.merge!(variant_unit:) if has_attribute?(:variant_unit)
+      units.merge!(variant_unit_name: '') if reset_variant_unit_name?
```

Comment:
Updating with the emtpy string as per Line#123 of the spec:

https://github.com/openfoodfoundation/openfoodnetwork/blob/7302c2d1618118995a73f9f9b467a00eff3fd92f/spec/system/admin/products_spec.rb#L95-L131